learn/teams/manage-users.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Manage Users
+id: ""
+---
+---
+
+The **Manage Users** page lets you view and manage all users in your team. Search by email address or add anybody by email address, whether they have a WaveMaker account or not. Those new to WaveMaker will receive an invite and get automatically added to your Team.
+
+You can access the Manage Users page in the administration area by clicking on the **Manage Users** entry in the left panel.
+
+![Manage users](/learn/assets/TeamManageUsers.png)
+
+From the user list, you can
+
+-  Create user or add existing user  
+-  View or edit a user’s role  
+-  Block a user, to prevent users from logging in to your team   
+-  Delete a user  
+
+### Creating Users
+
+As an admin user, you can create new users by clicking the **Add User** button.
+
+![Add users](/learn/assets/TeamAddUserButton.png)
+
+This will take you to the Add User Form where you fill out the new user’s details and click **Add** button.
+
+![Add users](/learn/assets/TeamAddUserForm.png)
+
+
+### Change User's Role
+
+You can change a single user’s role by using the **Change Role** icon located in each row in the table
+
+![Add users](/learn/assets/TeamUserChangeRole.png)
+
+### Block a User
+
+You can block/unblock a user by using the **block/ublock** icon located in each row in the table
+
+![Add users](/learn/assets/TeamUserBlock.png)
+
+
+### Remove a User
+
+You can delete a user by using the **Delete** icon located in each row in the table
+
```

Comment:
Usecase for Block and Delete ?
when to do what ?
what happens internally when click on Block and click on Delete